src/MCPClient/lib/clientScripts/normalizeReport.py

```diff
@@ -0,0 +1,248 @@
+#!/usr/bin/env python2
+
+from __future__ import print_function
+
+from argparse import ArgumentParser
+import logging
+from smtplib import SMTPException
+import sys
+import time
+
+import django
+django.setup()
+from django.contrib.auth.models import User
+from django.core.mail import send_mail
+from django.db import connection
+from django.template import Context, Template
+
+from main.models import Report
+
+from custom_handlers import get_script_logger
+
+
+logger = get_script_logger('archivematica.mcp.client.normalizeReport')
+
+# Based on http://leemunroe.github.io/responsive-html-email-template/email.html
+EMAIL_TEMPLATE = """
+    <!doctype html>
+    <html>
+        <head>
+            <meta name="viewport" content="width=device-width">
+            <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+            <title>Normalization failure report: {{ failed_count }} failures found</title>
+        </head>
+        <body>
+            <table cellpadding="10" width="100%">
+                <tr>
+                    <td></td>
+                    <td>
+                        <h1>Archivematica</h1>
+                        <hr />
+                        <h3>
+                            {{ name }}<br />
+                            Normalization failure report: {{ failed_count }} failures found.
+                        </h3>
+                        <table cellpadding="10" border="1" width="100%">
+                            <thead>
+                                <tr>
+                                    <th>File</th>
+                                    <th>Type</th>
+                                    <th>Exit code</th>
```

Comment:
When there's both an access and preservation error, this only displays the header for one